app.rb

```diff
@@ -0,0 +1,114 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
+  attr_accessor :person, :books, :rentals
+
+  def initialize()
+    @person = []
+    @books = []
+    @rentals = []
+  end
+
+  def create_book
+    puts ''
+    print 'Enter book title: '
+    title = gets.chomp
+    print 'Enter book author: '
+    author = gets.chomp
+    @books << Book.new(title, author)
+    puts 'Success'
+  end
+
+  def list_all_books
+    puts ''
+    @books.each { |book| puts "Title: #{book.title}", "Author: #{book.author}" }
+  end
+
+  def create_person
+    puts ''
+    print 'Enter 1 to add Student and 2 to add Teacher: '
+    selection = gets.chomp.to_i
+    print 'Enter name: '
+    name = gets.chomp
+    print 'Enter age: '
+    age = gets.chomp.to_i
+
+    case selection
+    when 1
+      print 'Does student have parent permission [Y/N]: '
+
+      permission = gets.chomp.downcase == 'y'
+
+      student = Student.new('sd', age, name, parent_permission: permission)
+      @person << student
+
+    when 2
+      print 'What is the teachers specialization: '
+      specialization = gets.chomp
+
+      teacher = Teacher.new(name, age, specialization, parent_permission: true)
+
+      @person << teacher
+    end
+
+    print 'Success'
```

Comment:
- [ ] Good job for creating a teacher ✨ However, the teacher's attributes are misplaced on listing all people. Please make sure to fix this ⛏

Current UI | Expected UI
---| ---|
![image](https://github.com/ragangithub/OOP_school_library/assets/106119105/5090deda-e63f-40d4-8340-118666c5b934) | ![image](https://github.com/ragangithub/OOP_school_library/assets/106119105/03cf8258-7150-41f7-8d26-f61f0837f8f9)